.azure-pipelines/frontend.yml

```diff
@@ -4,66 +4,65 @@ pool:
 trigger:
   branches:
     include:
-    - '*'
+      - '*'
   paths:
     include:
-    - .azure-pipelines/frontend.yml
-    - frontend/*
-    
+      - .azure-pipelines/frontend.yml
+      - frontend/*
+
 pr:
   branches:
     include:
-    - '*'
+      - '*'
   paths:
     include:
-    - .azure-pipelines/frontend.yml
-    - frontend/*
+      - .azure-pipelines/frontend.yml
+      - frontend/*
 
 steps:
-- task: NodeTool@0
-  inputs:
-    versionSpec: '10.x'
-  displayName: 'Install Node.js'
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
 
-- task: CacheBeta@0
-  inputs:
-    key: npm | $(Agent.OS) | frontend/package-lock.json
-    path: frontend/node_modules
-  displayName: Cache npm
+  - task: CacheBeta@0
+    inputs:
+      key: npm | $(Agent.OS) | frontend/package-lock.json
+      path: frontend/node_modules
+    displayName: Cache npm
 
-- script: |
-    npm install -g @angular/cli
-    npm install
-    cd ./node_modules/protractor && npm i webdriver-manager@latest && cd ../..
-    npm run pree2e
-  workingDirectory: frontend
-  displayName: 'npm install'
+  - script: |
+      npm install -g @angular/cli
+      npm install
+      cd ./node_modules/protractor && npm i webdriver-manager@latest && cd ../..
+      npm run pree2e
+    workingDirectory: frontend
+    displayName: 'npm install'
 
-- script: |
-    export CI=true
-    ./node_modules/tslint/bin/tslint -t json -o report.json --project . --force
-    ng test --watch=false --progress=false --code-coverage
-    ng e2e
```

Comment:
Why removed End To End Test? Please add it back.

Reply:
we removed it because it was causing a bug with webdriver and we don't have End To End tests

Reply:
We should have end to end tests, and not sweep under the rug the issue with webdriver. 
E2E tests are very useful to detect issue with angular templates compilation, dependencies injection, AOT... Could you please add a simple E2E test to test that the application load and is properly displayed?

Reply:
So far we have no e2e tests, and when we'll take time to write some, I think we'd rather use Cypress. So IMHO no need to waste our time trying to fix this protractor / webdriver issue.

Reply:
Ok, I'll close this comment and I will create another issue to add proper e2e tests.